README.md

```diff
@@ -17,53 +29,89 @@ or
 ```bash
 sudo npm install -g grunt-cli
 ```
-and to install all the other dependencies of the project, run in the root directory of the project
+
+3, Use NPM to install the remaining dependencies. In the root directory of the
+project run:
 ```bash
 npm install
 grunt bower
 ```
-Which first installs the node.js dependencies (grunt, bower, karma) and then runs the grunt task for installing the bower packages
 
-### Important! 
-only install bower packages through grunt, because grunt moves them to static/libs and appengine and karma both expect
-them to be there and not in bower_components also static/libs has a different structure then bower_components.
+This will install the node.js dependencies (grunt, bower, karma) and then run
+the grunt task for installing the bower packages.
+
+### Important!
+The bower packages should only be installed using grunt, since grunt will place
+them in static/libs, where both appengine & karma expect them. Calling bower
+directly will install them in bower_components.
 
-## Run unit-tests for rdm.js
+## Common Tasks
 
-Currently only rdm.js in static/js has a javascript unit-test using [karma](https://github.com/karma-runner/karma) the unit-test is
-located in unit-test-js/tests/rdm.js and the karma configuration in unit-test-js/karma.conf.js the test can be run by running
+Tasks are managed using Grunt. To see a list of tasks run:
+
+```bash
+grunt --help
+```
+
+### Run the Unit Tests
+
+The newer Angular code is unit-tested using
+[karma](https://github.com/karma-runner/karma). The tests are in
+unit-test-js/tests/rdm.js and the karma configuration in
+unit-test-js/karma.conf.js. The test can be run with:
 ```bash
 grunt unit-test
 ```
-It does require firefox and the dependencies to be installed to be able to run the test
+It does require firefox and the dependencies to be installed to be able to run
```

Comment:
yes it does, a GUI free option is [phantomjs](http://phantomjs.org/) it's karma runner is [here](https://github.com/karma-runner/karma-phantomjs-launcher)